README.md

```diff
@@ -398,37 +450,13 @@ exit /b 0
 popd
 echo Failed.
 exit /b 1
+```
 
-License of Minetest textures and sounds
----------------------------------------
-
-This applies to textures and sounds contained in the main Minetest
-distribution.
-
-Attribution-ShareAlike 3.0 Unported (CC BY-SA 3.0)
-http://creativecommons.org/licenses/by-sa/3.0/
-
-Authors of media files
------------------------
-Everything not listed in here:
-Copyright (C) 2010-2012 celeron55, Perttu Ahola <celeron55@gmail.com>
-
-ShadowNinja:
-  textures/base/pack/smoke_puff.png
-
-Paramat:
-  textures/base/pack/menu_header.png
+# Licensing
```

Comment:
This should be level 2 heading, and all headers logically "below" this one should be one more level.